inc/common/purge.php

```diff
@@ -527,3 +527,19 @@ function rocket_clean_cache_theme_update( $wp_upgrader, $hook_extra ) {
 
 	rocket_clean_domain();
 }
+
+/**
+ * Purge WP Rocket cache on Slug / Permalink change.
+ *
+ * @since  3.4.2
+ * @author Soponar Cristina
+ *
+ * @param int   $post_id   The post ID.
+ * @param array $post_data Array of unslashed post data.
+ */
+function rocket_clean_cache_on_slug_change( $post_id, $post_data) {
```

Comment:
```suggestion
function rocket_clean_post_cache_on_slug_change( $post_id, $post_data ) {
```

I think adding `post` in the function name would be better to clearly identify it's clearing only this post cache